### ISSUE TITLE
Support pictures on the left side of the notification instead of on the right, where attachments appear

### ISSUE THREAD

**Issue by rocboronat** [MaikuB/flutter_local_notifications#1544]
Right now, on iOS, the only way we have to show a picture is to add it as an attachment. That's not bad because it just works, but Telegram and WhatsApp show the sender's picture on the left of the notification instead of on the right, where the attachments appear. That makes our app feel different, and it makes the UX slightly worse. You know, the users wonder "Why these app notifications are different from Telegram's and WhatsApp's if they should seem the same?"

Here's how WhatsApp and Telegram do it:

<img src="https://user-images.githubusercontent.com/456499/160576886-aa85a3a1-8d27-42b5-b928-2b43ec17bed5.jpg" width=400></img>

And here's the best thing I can do by using this plugin:

<img src="https://user-images.githubusercontent.com/456499/160577002-5895fa8b-7674-4568-a6e4-b42ba04d76fa.png" width=400></img>

By the way, thank you so much for this plugin and your support 👍

**Comment by MaikuB** [comment]
Whilst you may have examples of different layouts, this plugin's responsibility is to be a wrapper for native APIs. Have you actually looked at the native APIs though (e.g. look at Apple's documentation)? This should be the starting point and understanding what is provided by native APIs is something you should know as part of developing your apps to assess technical feasibility etc where you may be required to even use the native APIs yourself (i.e. without using a plugin)

**Comment by rocboronat** [comment]
Hi again, and thanks for the answer! Do you mean WhatsApp and Telegram's notifications are not achiveable using the regular iOS notifications API but using custom notifications?

**Comment by MaikuB** [comment]
No I'm not but may well be the case given a Google search for "ios notification image position" has results that point to custom notifications. If that is what's needed then you would need to deal with this yourself. Whilst I maintain a notifications plugin, it doesn't mean I'm an expert in this area to provide all the answers and this is something I do in my limited, spare time. With that in mind and the points I raised in my previous post, what I wrote was really a question on if you have actually looked into Apple's notification APIs/functionality to understand what is required. The API docs I had put in for the notification attachments had a link to the native notification class (see https://developer.apple.com/documentation/usernotifications/unnotificationattachment?language=objc) and there's no mention
